extensions/cornerstone-dicom-sr/src/utils/getRenderableData.ts

```diff
@@ -145,13 +100,19 @@ function _getRenderableData(GraphicType, GraphicData, imageId) {
       // and then choose the correct points to use for the ellipse.
 
       const pointsWorld: csTypes.Point3[] = [];
-      for (let i = 0; i < GraphicData.length; i += 2) {
-        const worldPos = utilities.imageToWorldCoords(imageId, [
-          GraphicData[i],
-          GraphicData[i + 1],
-        ]);
 
-        pointsWorld.push(worldPos);
+      if (ValueType === 'SCOORD3D') {
```

Comment:
This looks like the right way to convert this data - it will work for 3d length and other 3d coordinates.